src/pmhn/_trees/plot_all_mut_freq.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
```

Comment:
I think `plot_all_mut_freq.py`, `plot_all_trees.py` and `write_csv.py` are nice scripts, but they should go to the `warmup` directory. 
(Namely, `src/pmhn/_trees` is a Python package, think NumPy or Matplotlib, and we want to separate it from scripts and experiments)